### ISSUE TITLE
Maybe false positive ? - VIP.ValidatedSanitizedInput

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#190]
The below snippet triggers the `Detected usage of a non-sanitized input variable: $_POST` error, while in this context I would expect it to be acceptable as the validation/sanitation is/should be done within `wp_nonce_verify()`.

``` php
    if ( ! wp_verify_nonce( $_POST['nonce'], 'my_secret_nonce' ) ) {
        die();
    }
```
